app/models/game.rb

```diff
@@ -2,7 +2,11 @@ class Game < ActiveRecord::Base
   
   scope :is_available, -> { where("black_player_id is null or white_player_id = 0") }
   
-  after_create :populate_board!
+  after_create do
```

Comment:
It's best to not make modifications to non-test code in order to allow for testing. This is called Test Induced Design Damage.

Why not instead add an `after(:build)` in the game to skip the `:create`, `:after` callback? You could even make a subfactory to allow it `:game_with_pieces` or something